### ISSUE TITLE
SSR not working due to window check

### ISSUE THREAD

**Issue by nestarz** [haldarmahesh/use-key-hook#15]
I get WebpackError: ReferenceError: window is not defined when doing SSR, how can I avoid it ? Thanks

**Comment by haldarmahesh** [comment]
Hey @nestarz thanks for raising this.
Can you please give little more details of webpack and project setup so that I can replicate the same?
Thanks

**Comment by jamesmosier** [comment]
I think a potential solution would be for checking to ensure `window` is defined before setting the event listeners. I personally use [this package](https://github.com/akiran/can-use-dom) but the contents of that package are pretty straightforward if you don't want to include another dependency. 

```
var canUseDOM = !!(
  typeof window !== 'undefined' &&
  window.document &&
  window.document.createElement
);
```

**Comment by haldarmahesh** [comment]
@nestarz thanks for reporting this issue.
@jamesmosier thanks for putting your view.
This has been fixed in `v1.5.0`, please upgrade and you should not face this issue again.
Cheers.
